Makefile

```diff
@@ -103,6 +102,18 @@ firebase-public: frontend
 
 .PHONY: firebase-deploy firebase-deploy-prod firebase-emulate firebase-public
 
+# --- scripts ------------------------------------------------------------------
+SCIRPTS = firebase/deploy
+
+sh-lint: ## Lint script files with shellcheck and shfmt
+	shellcheck $(SCIRPTS)
+	shfmt --indent 4 --list $(SCIRPTS)
```

Comment:
a little odd that `sh-lint` will format the files - usually you wouldn't expect a lint target to change the files.

Reply:
It doesn't. It errors if it isn't formatted and litst the files that aren't.
It's a bit like the gofumpt linter i think that yells if your file is not gofumpted.